bokeh/plotting/figure.py

```diff
@@ -49,6 +66,91 @@ def __init__(self, *arg, **kw):
         tool_objs = _process_tools_arg(self, tools)
         self.add_tools(*tool_objs)
 
+        if title:
+            title = Title(
+                title,
+                text_font=title_text_font,
+                text_font_size=title_text_font_size,
+                text_font_style=title_text_font_style,
+                text_color=title_text_color,
+                text_alpha=title_text_alpha,
+                text_align=title_text_align,
+                text_baseline=title_text_baseline
```

Comment:
suggest just putting all the `kw.pop`'s inline here instead of creating all these variables that are not used anywhere else